app/components/tetris/index.gts

```diff
@@ -43,9 +44,17 @@ export class Tetris extends Component<TetrisSignature> {
     this.tetrominoSequence,
   );
 
+  constructor() {
```

Comment:
Hier musst du vermutlich (owner: Tetris, args: TetrisSignature['Args']) definieren und dann  super(owner, args); verwneden.

Reply:
Ich hab das von hier übernommen: https://docs.ember-cli-typescript.com/ts/using-ts-effectively#class-property-setup-errors

> Assertion Failed: Attempting to lookup an injected property on an object without a container, ensure that the object was instantiated via a container. – You failed to pass `...arguments` when you called `super` in e.g. a component class `constructor`. Always do `super(...arguments)`, not just `super()`, in your constructor.

Reply:
Der Fehler ist:

> TS2556: A spread argument must either have a tuple type or be passed to a rest parameter.

Reply:
Hab es jetzt wie hier beschrieben gemacht: https://docs.ember-cli-typescript.com/ember/components#adding-arguments-and-giving-them-a-type